app/models/ministerial_role.rb

```diff
@@ -27,6 +30,18 @@ def self.cabinet
     where(cabinet_member: true).alphabetical_by_person
   end
 
+  def self.ministerial_roles_with_current_appointments
+    includes(:translations).cabinet_members.order(:seniority).joins(:current_role_appointments)
```

Comment:
I did see this [Sentry error](https://govuk.sentry.io/issues/5529352718/?referrer=slack&notification_uuid=d2446203-0e50-4fca-a483-d2ea78e13e3b&alert_rule_id=963054&alert_type=issue) appear when your branch was deployed - is there a further tweak required to this PR so that [this bit](https://github.com/alphagov/whitehall/blob/49b6515a11cae20199c4d0b03d33fd0f07610099/app/presenters/publishing_api/ministers_index_presenter.rb#L64-L70) doesn't raise an exception?